x/stakeibc/keeper/msg_server_liquid_stake.go

```diff
@@ -19,10 +19,10 @@ func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake)
 	// check that hostZone is registered
 	// strided tx stakeibc liquid-stake 100 uatom
 	hostDenom := msg.Denom
-	hostZone, hostZoneFound := k.GetHostZoneFromDenom(ctx, msg.Denom)
-	if !hostZoneFound {
+	hostZone, err := k.GetHostZoneFromHostDenom(ctx, msg.Denom)
```

Comment:
Can we rename this Denom (msg.Denom)? I know we have it in the backlog

Reply:
Done. Updated `tx.proto`, the type files and keeper functions. Also updated denom on `MsgRedeemStake` to `StAssetDenom`.